src/op_mode/dhcp.py

```diff
@@ -279,22 +284,36 @@ def _get_raw_server_static_mappings(family='inet', pool=None, sorted=None):
 
     if sorted:
         if sorted == 'ip':
-            data.sort(key = lambda x:ip_address(x['ip-address']))
+            if family == 'inet6':
+                mappings.sort(key = lambda x:ip_address(x['ipv6-address']))
+            else:
+                mappings.sort(key = lambda x:ip_address(x['ip-address']))
         else:
-            data.sort(key = lambda x:x[sorted])
+            mappings.sort(key = lambda x:x[sorted])
     return mappings
 
 def _get_formatted_server_static_mappings(raw_data, family='inet'):
     data_entries = []
-    for entry in raw_data:
-        pool = entry.get('pool')
-        subnet = entry.get('subnet')
-        name = entry.get('name')
-        ip_addr = entry.get('ip-address', 'N/A')
-        mac_addr = entry.get('mac', 'N/A')
-        duid = entry.get('duid', 'N/A')
-        description = entry.get('description', 'N/A')
-        data_entries.append([pool, subnet, name, ip_addr, mac_addr, duid, description])
+    if family == 'inet':
+        for entry in raw_data:
+            pool = entry.get('pool')
+            subnet = entry.get('subnet')
+            name = entry.get('name')
+            ip_addr = entry.get('ip-address', 'N/A')
+            mac_addr = entry.get('mac', 'N/A')
+            duid = entry.get('duid', 'N/A')
+            description = entry.get('description', 'N/A')
+            data_entries.append([pool, subnet, name, ip_addr, mac_addr, duid, description])
+    elif family == 'inet6':
```

Comment:
Instead of duplicating all these lines for the one `ipv6-address` key difference. Could this instead be handled by:
`ip_addr = entry.get('ipv6-address' if family == 'inet6' else 'ip-address', 'N/A')`

Reply:
Of course, normally i'll be doing so, but most part of handling code inside the same file has been split for the two inet familly

Reply:
> Could this instead be handled by: ip_addr = entry.get('ipv6-address' if family == 'inet6' else 'ip-address', 'N/A')

We need room for future improvements, too ;)